docs/product-assembly_business-optimizer/src/main/asciidoc/main.adoc

```diff
@@ -16,8 +17,11 @@ Use this document to set up and configure {PLANNER} functionality.
 The *Employee Rostering* sample project has been created in {CENTRAL} to demonstrate Business Optimizer capabilities. To view this project, you must have {PRODUCT} installed. For more information about setting up and installing {PRODUCT} and {CENTRAL}, see _Getting started with decision services_.
 
 // Modules
-
+//About
 include::product-business-resource-planner-guide/optimizer-about-business-optimizer-con.adoc[leveloffset=+1]
+include::product-business-resource-planner-guide/optimizer-planning-problem-con.adoc[leveloffset=+1]
+
+//Downloading and installing
 include::product-business-resource-planner-guide/optimizer-download-proc.adoc[leveloffset=+1]
 include::product-business-resource-planner-guide/optimizer-engine-installation-proc.adoc[leveloffset=+1]
```

Comment:
And again since this file isn't part of the PR, just mentioning that the context statement following the header should provide more of a statement of the why and wherefore, i.e., it's context, rather than a command. Like you've done well elsewhere. And shouldn't end in a colon.

Something as simple as (just my whack at it):
= Installing the {PLANNER} engine in your application
You can install {PLANNER} with Maven and with various other applications, such as Gradle, Ivy, or Buildr, to optimize your existing planning environment.

Or something. I would assess all context statements, especially in procedures (the biggest pitfall) to make sure they are genuine context statements. Here's the [templates page](https://mojo.redhat.com/docs/DOC-1099486#jive_content_id_Master_source_files_in_asciidoc_format) too, if you need it.

Also, just a note that the prereq for this section ("Get the {PLANNER} JARs ...") is an example of a prereq that could arguably just be the first step in the actual procedure. The argument, again, for passive prereqs is that if you can't make it passive and sound normal (where active is really the only way) then it's highly likely it should be part of the procedure. I don't know for sure in this case. Just sayin'. Your call.

Lastly, the "Installing with Ant" sub-heading should be it's only module, included in the assembly. And the context statement should likewise be updated like I've described above.

Reply:
I'm reluctant to break Ant into it's own module as it's a very niche use case. This will *never* be reused anywhere else. I'd prefer to maybe reformat into some kind of alternative step, as it's still installing the engine, it's just using a different application than 99% of users would use.